libcloud/compute/drivers/gce.py

```diff
@@ -801,7 +802,8 @@ def __init__(self, user_id, key=None, datacenter=None, project=None,
         else:
             self.region = None
 
-    def ex_list_disktypes(self, zone=None):
+    def ex_list_disktypes(self, zone=None, ex_filter=None, ex_max_results=None,
+                          ex_reset_page_token=False):
```

Comment:
If the method already has an `ex_` prefix, the argument shouldn't have it.

The arguments only need to have `ex_` prefix if the arguments are non-standard and used with a standard method (e.g. list_nodes, etc.).